security/s2a/internal/crypter/common.go

```diff
@@ -7,6 +7,10 @@ const (
 	// nonceSize is the size of the nonce in number of bytes for
 	// AES-128-GCM-SHA256, AES-256-GCM-SHA384, and CHACHA20-POLY1305-SHA256.
 	nonceSize = 12
+	// sha256DigestLength is the digest length of sha256 in bytes.
+	sha256DigestLength = 32
```

Comment:
Can we use `Size` rather than `Length`? It seems we're using `Size` everywhere else.

Reply:
I don't think this was addressed...